laser_scan_matcher/src/laser_scan_matcher.cpp

```diff
@@ -162,6 +166,12 @@ void LaserScanMatcher::initParams()
   if (!nh_private_.getParam ("use_vel", use_vel_))
     use_vel_ = false;
 
+  // **** Are velocity input messages stamped?
+  // if false, will subscribe to Twist msgs on /vel
+  // if true, will subscribe to TwistStamped msgs on /vel
+  if (!nh_private_.getParam ("stamped_vel", stamped_vel_))
+    stamped_vel_= true;
```

Comment:
I think I'm right, so I added [this change](https://github.com/ccny-ros-pkg/scan_tools/pull/35/files#diff-a3dc78e10ccafc48407c63f7e365da34R173) in #35.